postfix

```diff
@@ -30,6 +30,9 @@ postconf -e "smtp_sasl_password_maps = static:${SMTP_SASL_PASSWORD_MAPS}"
 ## validate SPF
 sed -i '/postlog/a\\policy-spf  unix  -       n       n       -       -       spawn \n user=nobody argv=/usr/bin/postfix-policyd-spf-perl' /etc/postfix/master.cf
 
+## Adding Gmail style ( ignoring dots). 
+echo "/^(.*)\.(.*)@((.*\.)*UW\.PARTNERS)$/ ${1}${2}@${3}" > /etc/postfix/canonical
```

Comment:
If you know this works and you don't want to spend the time on it - just merge it in.

But if you want a review:

1. Why is domain capitalised?
2. Current regex will only drop the last dot - is that correct assumed behaviour, or do we want to support dropping all dots.
    ```
    george.angel.something@uw.partners
    ### after replacement
    george.angelsomething@uw.partners
    ```

Reply:
We'll want to drop all dots

Reply:
https://superuser.com/a/1423791 implies that this should have matched all dots

Reply:
Yes - but the reality:

```
smarthost-smtpd-relayhost-79d749767c-9ftst smarthost Mar 24 09:45:06 email postfix/error[172]: DCE8C1FD84A: to=<gangel.dots.test@>, orig_to=<gangel.dot
s.test@dev.uw.partners>, relay=none, delay=0.25, delays=0.25/0/0/0, dsn=5.1.3, status=bounced (bad address syntax)
```

Reply:
Ah 😞